File: programme/sum-of-cubes/SumOfCube.java
// Write a programme to find sum of cubes of first n natural numbers.
import java.util.*;
public class SumOfCube {
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        System.out.print("Input: ");
        int n = sc.nextInt();
        int total = 0;
        for(int i=1;i<=n;i++){
            total += (i*i*i);
        }
        System.out.println("Output: "+total);
        sc.close();
    }
}

File: programme/sum-of-cubes/sum-of-cube.js
/* 
Write a programme to find sum of cubes of first n natural numbers.
*/

function sumOfCube(num) {
  const numsArr = [...new Array(num + 1).keys()].slice(1);
  return numsArr.reduce((acc, curr) => acc + curr ** 3, 0);
}

const num = 3
console.log(`The sum of cube for ${num}: ${sumOfCube(num)}`)